### DISCUSSION TITLE
Look what we did with Emscripten

### DISCUSSION THREAD

**Discussion by aharondavid** [emscripten-core/emscripten#18499] [Show and tell]
Our team has successfully ported our desktop application, consisting of 500,000 lines of C/C++ code and multiple third-party library dependencies, to a web-based platform! Check out the results here:
https://www.linkedin.com/feed/update/urn:li:activity:7021487629019803648/

A big thank you to Emscripten and its team, particularly **Alon Zakai** for his amazing initiative, the developer community for their support, and special thanks to [Sam Clegg](https://github.com/sbc100) for providing prompt and incredibly helpful responses to our questions throughout the process.

**Comment by sbc100** [comment]
Great achievement!    I enjoyed zooming all the way in to see my condo in SF!

**Comment by svenpilz** [comment]
And always nice to see OSM data in the wild :).

**Reply by sasmaster** [reply]
Looks huge. Did you experience any performance issues during the porting?

**Reply by aharondavid** [reply]
No, the performance is very close to native.

**Comment by allan-bonadio** [comment]
Very cool!  Wish I'd done it.
